Footer

ConstantineFrol opened the pull request:
First pull

ConstantineFrol committed:
issue#1

create footer

ConstantineFrol committed:
issues#2

Inject Footer

ConstantineFrol committed:
Issues#3

ModuleNotFound error fix

ConstantineFrol committed:
issue#4

Change in footer(added router for links)

ConstantineFrol committed:
issue#5

added react-router-dom

ivarprudnikov commented on src/auth/Components/About.js:
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+export default function About() {
```
Move this to `pages/about.js`

ivarprudnikov commented on src/auth/Components/Cookies.js:
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+export default function Cookies() {
```
Move this to `pages/cookies.js`

ivarprudnikov commented on src/auth/Components/Privacy.js:
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+export default function Privacy() {
```
Move this to `pages/privacy.js`

ivarprudnikov commented on src/auth/Components/TermOfUse.js:
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+export default function TermOfUse() {
```
Move this to `pages/terms.js`

ivarprudnikov commented on pages/_app.js:
```diff
@@ -1,9 +1,11 @@
 import 'bootstrap/scss/bootstrap.scss'
 import { AuthContextProvider } from '../src/auth/AuthContextProvider'
+import {Footer} from '../src/auth/Components/Footer';
```
There is no need to import it here

ivarprudnikov commented on package.json:
```diff
@@ -11,7 +11,9 @@
     "bootstrap": "^4.6.0",
     "next": "^10.0.8",
     "react": "^17.0.1",
+    "react-bootstrap-validation": "^0.1.11",
```
remove this as it is not related here

ivarprudnikov commented on package.json:
```diff
@@ -11,7 +11,9 @@
     "react": "^17.0.1",
+    "react-bootstrap-validation": "^0.1.11",
     "react-dom": "^17.0.1",
+    "react-router-dom": "^5.2.0",
```
Not necessary here as well, please remove. See more comments in Footer.js

ivarprudnikov commented on src/auth/Components/Footer.js:
```diff
@@ -0,0 +1,42 @@
+
+const Footer = () =>(
+
+  <Container fluid style={{backgroundColor: '#212529', color: '#fff'}}>
```
Here you need to use `next/link` instead of the react router. When each of the pages are moved to `pages/` directory, you will be able to link to them just like it is described in https://nextjs.org/docs/api-reference/next/link

ConstantineFrol committed:
issue#6

change from router to next-link, moved pages (Term of use, About, Cookies, Privacy) to './pages', remove un used imports from 'package.json'

ivarprudnikov commented on pages/Footer.js:
```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import Link from 'next/link';
+
+const Footer = () =>(
```
Please move this to `src` directory as it is not a page but rather a shared component

ivarprudnikov commented on pages/About.js:
```diff
@@ -0,0 +1,9 @@
+export default function About() {
+  return (
+    <div>
+      <h1>About Page</h1>
```
There is no footer here, how to navigate back?

ivarprudnikov commented on pages/Cookies.js:
```diff
@@ -0,0 +1,9 @@
+export default function Cookies() {
+  return (
+    <div>
+      <h1>Cookies term Page</h1>
```
There is no footer here, how to navigate back?

ivarprudnikov commented on pages/Privacy.js:
```diff
@@ -0,0 +1,9 @@
+export default function Privacy() {
+  return (
+    <div>
+      <h1>Privacy term Page</h1>
```
There is no footer here, how to navigate back?

ivarprudnikov commented on pages/TermOfUse.js:
```diff
@@ -0,0 +1,9 @@
+export default function TermOfUse() {
+  return (
+    <div>
+      <h1>Term of Use Page</h1>
```
There is no footer here, how to navigate back?

ivarprudnikov commented on pages/Footer.js:
```diff
@@ -0,0 +1,38 @@
+
+  <Container fluid style={{backgroundColor: '#212529', color: '#fff'}}>
+    <Container>
+    <ul>
```
Would it be possible to use styling from `base-nav` example https://getbootstrap.com/docs/4.6/components/navs/#base-nav

ivarprudnikov commented on pages/Footer.js:
```diff
@@ -0,0 +1,38 @@
+
+const Footer = () =>(
+
+  <Container fluid style={{backgroundColor: '#212529', color: '#fff'}}>
```
Where does `Container` come from?

ivarprudnikov reviewed (changes requested):
There are some comments here but also please discard the `package-lock.json` changes as there are no new dependencies.

ConstantineFrol committed:
issue#7

Fixed links

ConstantineFrol committed:
issue#8

delete styles folder

ivarprudnikov commented on pages/index.js:
```diff
@@ -1,6 +1,6 @@
 import Link from 'next/link'
 import { useAuthContext } from '../src/auth/AuthContextProvider'
-import Footer from './Footer';
+import Footer from '../src/auth/Footer';
```
move footer to `/src` and out of `auth` folder as it is not part of the authentication

ivarprudnikov commented on package-lock.json:
```diff
@@ -1,8 +1,4433 @@
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
```
You need to discard all the  `package-lock.json` changes, these cannot be in this PR as there are no new dependencies. Please remove those by resetting your `package-lock.json` to the version which is on the `master branch`.

ConstantineFrol committed:
issue#9

moved Footer to src, update package-lock.json

ivarprudnikov committed:
Merge branch 'master' into footer

ivarprudnikov committed:
add some styling to footer

ivarprudnikov committed:
use flex and stretch main content
